packages/client/auth/src/CrossmintAuthClient.ts

```diff
@@ -232,7 +232,10 @@ export class CrossmintAuthClient extends CrossmintAuth {
 
     public async authenticateSmartWallet(address: string, signature: string) {
         try {
-            const queryParams = new URLSearchParams({ signinAuthenticationMethod: "evm" });
+            const queryParams = new URLSearchParams({
+                signinAuthenticationMethod: "evm",
+                callbackUrl: `${this.apiClient.baseUrl}/${AUTH_SDK_ROOT_ENDPOINT}/callback`,
```

Comment:
our backend still needs this callbackUrl (doesn't actually do anything anymore) but fixes an error we get client side.